Expired instance - final docs update (#1272)
kyma-project/kyma-environment-broker: docs/contributor/03-30-trial-and-free-expiration.md

{
	“service_id”: “47c9dcbf-ff30-448e-ab36-d3bad66ba281", //Kyma ID
	“context”: {
		“globalaccount_id”: “{GLOBALACCOUNT_ID}”
	}
}
```

> Requests that don't include **globalaccount_id** or it is empty, then KEB return the HTTP response `400`.

Reviewer: ```suggestion
For requests that don't include **globalaccount_id** or are empty, KEB returns the HTTP response `400`.
```
